### PR TITLE
ref(SSRCValidator): improvements

### PR DESCRIPTION
**Pull request by paweldomas**
Gets rid of media type where video specific groups are in use. Other minor refactoring.

### CODE REVIEW

File: src/main/java/org/jitsi/protocol/xmpp/util/SSRCSignaling.java [hunk 1]
```diff
@@ -362,6 +419,11 @@ public static boolean mergeVideoIntoAudio(MediaSourceMap peerSSRCs)
             throw new IllegalArgumentException("Null or empty 'groupMsid'");
         }
 
+        // FIXME change once migrated to Java 8
+        // return groups.stream()
+        //     .filter(group -> group.getGroupMsid().equalsIgnoreCase(groupMsid)
+        //     .collect(Collectors.toList());
+
```
Comment (bbaldino) [inline src/main/java/org/jitsi/protocol/xmpp/util/SSRCSignaling.java]: i think we can just leave this out

Comment (paweldomas) [reply src/main/java/org/jitsi/protocol/xmpp/util/SSRCSignaling.java]: no, lets keep it

**Comment by bgrozev** [comment]
Should we merge this?

**Comment by bbaldino** [comment]
jenkins, test this please

**Pull request merged**